modules/build-raptor/src/build-raptor-cli.ts

```diff
@@ -81,15 +130,19 @@ async function run(options: Options) {
       return
     }
 
-    if (!commitHash) {
-      throw new Error(`missing commit hash in CI`)
+    const endpointData = { packageName: u.id, commitHash, pullRequest, casReference: resolved }
+    const parsedRequest = RegisterAssetRequest.safeParse(endpointData)
+
+    if (!parsedRequest.success) {
+      throw new Error(`Invalid request data: ${endpointData}`)
```

Comment:
doing a `.safeParse()` and then throwing an exception if it's unsuccessful is pretty much equivalent to doing a `.parse()`